documentation/en/source/pages/command-line/scan.rst

```diff
@@ -56,10 +56,10 @@ Executes a deep scan of a running system.
 
 	* ``--ip`` (mandatory): the IP address or fully qualified hostname of the running system
 	* ``--scan-login`` (mandatory): the root user name (normally root)
-	* ``--name`` (mandatory): the scan name to use when creating the scan meta-data
+	* ``--name`` (mandatory): the scan name to use when creating the scan meta-data. If you want to use spaces in the scan name, put it inside quotes and double quotes like this: ``--name '"my scan name with spaces"'``
```

Comment:
If you ...., you must use single quote and double quotes before and after the name. For example: